css-coding-conventions.md

```diff
@@ -0,0 +1,7 @@
+# CSS Coding Conventions
```

Comment:
I noticed we split up our How-to oan PRs in a separate file and thought it would be good to split up this part as well. If you think it should all be in the read me I am fine with that.

Reply:
@ThijsTyZ do you want to include the documentation as a chapter in the README? Then I will update the PR with this change

Reply:
I think we should include this inside the `README.md`. The proposal of @psimk should be handled in a different PR.

Reply:
issue for that proposal: https://github.com/mediamonks/frontend-coding-standards/issues/84